packages/app/firebase-schema.json

```
@@ -81,6 +81,10 @@
         "android_task_executor_keep_alive_seconds": {
           "description": "Keep-alive time of ThreadPoolExecutor used by RNFirebase for Android, in seconds. Defaults to `3`.\n Excess threads in the pool executor will be terminated if they have been idle for more than the keep-alive time.",
           "type": "number"
+        },
+        "background_activity_names": {
```

suggest making key android-specific
```suggestion
        "android_background_activity_names": {
```